agent/agent.go

```diff
@@ -684,7 +684,6 @@ func (a *Agent) runPreAndMain(ctx context.Context, tc *taskContext) (status stri
 
 	idleTimeoutCtx, idleTimeoutCancel := context.WithCancel(timeoutWatcherCtx)
 	go a.startIdleTimeoutWatcher(timeoutWatcherCtx, idleTimeoutCancel, tc)
-
```

Comment:
mega nit: can we keep the newline? There's a lot going on in this particular couple lines of code with starting up the background timeout watchers.